docs/src/literate-tutorials/dg_heat_equation.jl

```diff
@@ -0,0 +1,382 @@
+            for j in 1:n_basefuncs
+                ∇u = shape_gradient(cellvalues, q_point, j)
+                ## Add contribution to Ke
+                Ke[i, j] += (∇δu ⋅ ∇u) * dΩ
+            end
+        end
+    end
+    return Ke, fe
+end
+
+function assemble_interface!(Ki::Matrix, iv::InterfaceValues)
+    ## Reset to 0
+    fill!(Ki, 0)
+    ## Loop over quadrature points
+    for q_point in 1:getnquadpoints(iv)
+        ## Get the normal to face A
+        normal = getnormal(iv, q_point)
+        ## Get the quadrature weight
+        dΓ = getdetJdV(iv, q_point)
+        ## Loop over test shape functions
+        for i in 1:getnbasefunctions(iv)
+            ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+            test_jump = shape_value_jump(iv, q_point, i) * normal
+            test_grad_avg = shape_gradient_average(iv, q_point, i)
+            ## Loop over trial shape functions
+            for j in 1:getnbasefunctions(iv)
+                ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+                trial_jump = shape_value_jump(iv, q_point, j) * normal
+                trial_grad_avg = shape_gradient_average(iv, q_point, j)          
+                ## Add contribution to Ki          
+                order = Ferrite.getorder(iv.face_values_a.func_interp)
+                dim = Ferrite.getdim(iv.face_values_a.func_interp)
+                Ki[i, j] += -(test_jump ⋅ trial_grad_avg + test_grad_avg ⋅ trial_jump) * dΓ + (1 + order)^dim * 20/2 * (test_jump ⋅ trial_jump) * dΓ
+            end
+        end
+    end
+    return Ki
+end
+
+function assemble_boundary!(Ke::Matrix, fv::FaceValues)
+    ## Reset to 0
+    fill!(Ke, 0)
+    ## Loop over quadrature points
+    for q_point in 1:getnquadpoints(fv)
+        ## Get the normal to face A
+        normal = getnormal(fv, q_point)
+        ## Get the quadrature weight
+        dΓ = getdetJdV(fv, q_point)
+        ## Loop over test shape functions
+        for i in 1:getnbasefunctions(fv)
+            ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+            test_jump = shape_value(fv, q_point, i) * normal
+            test_grad_avg = shape_gradient(fv, q_point, i)
+            ## Loop over trial shape functions
+            for j in 1:getnbasefunctions(fv)
+                ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+                trial_jump = shape_value(fv, q_point, j) * normal
+                trial_grad_avg = shape_gradient(fv, q_point, j) 
+                order = Ferrite.getorder(fv.func_interp)
+                dim = Ferrite.getdim(fv.func_interp)
+                ## Add contribution to Ki                             
+                Ke[i, j] += -(test_jump ⋅ trial_grad_avg + test_grad_avg ⋅ trial_jump) * dΓ + (1 + order)^dim * 20/2 * (test_jump ⋅ trial_jump) * dΓ
```

Comment:
Can we have two Neumann boundaries for the example, so this term does not vanish?